packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CollectorExporterBase } from '../../CollectorExporterBase';
+import { CollectorExporterConfigBrowser } from './types';
+import * as collectorTypes from '../../types';
+import { parseHeaders } from '../../util';
+import { sendWithBeacon, sendWithXhr } from './util';
+
+/**
+ * Collector Metric Exporter abstract base class
+ */
+export abstract class CollectorExporterBrowserBase<
+  ExportItem,
+  ServiceRequest
+> extends CollectorExporterBase<
+  CollectorExporterConfigBrowser,
+  ExportItem,
+  ServiceRequest
+> {
+  _headers: Record<string, string>;
+  _useXHR: boolean = false;
```

Comment:
These should both be private
```suggestion
  private _headers: Record<string, string>;
  private _useXHR: boolean = false;
```